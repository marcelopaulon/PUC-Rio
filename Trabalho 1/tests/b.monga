/* Retirado de: https://github.com/jpfabris/inf1715/blob/8aed4f49d5a9651d8180fdb3d5e05e4d4db40929/trab1/Teste.txt */
int soma(int a,int b){ /*Funcao que soma dois inteiros*/
  return a+b;
}

int main(void){
  io = 0xff;
  int p1=10;
  int p2=p1*p1;
  int a=p2-p1;
  int b=p1*p2;
  p1=b/p2;
  p1 = soma(p1,p2);
  return p1;
}

float ret_float(void){
  /*****Retornando****/
  float a = 0.5;
  float b = 0xFD + 0xA1;
  float _c_ = a + b;
  float _2d = c/2;
  return _2d/2d; /*NÃ£o considera 2d um id*/
}

void main(){
  char a=new char[20];
  a="Teste Lexico";
  a="Teste com string";
  int b=10;
  int c=25;
  if (c > b){
     b=b*3;
  }
  else {
    b = c;
  }
  while (c >= 0){
    if(c && b && b == 1){
     	  c=c-1;
    }
  }
}

/***/
/*Testando*/
/**/a/****/
/***/a/****/
/**/*/a/****/
/**/a/**** **/
/***/** ** **/a/****/
/****/a/*******/***/*b*/

void main(){
  char a=new char[20];
  a="Testando primeira \\ string";
  a="Testando em \n duas linhas";
  a="\n Segunda linha";
  a="\n\"Entre aspas\". Tres aspas: \"\"\""
  a="Primeira \n Segunda \n Terceira";
  a="Testando\ttab1\ttab2";
  a="Testando\t\t\t 3 tabs";
  a="/***/";
  a="/****/";
}

void compare(int a, int b) {
  if(a >= b || b <= a) {
    a = a + 1;
  }
}
