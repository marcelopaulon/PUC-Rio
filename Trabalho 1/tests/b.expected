START
264(2)
269(2)soma
40(2)
264(2)
269(2)a
44(2)
264(2)
269(2)b
41(2)
123(2)
266(3)
269(3)a
43(3)
269(3)b
59(3)
125(4)
264(6)
269(6)main
40(6)
267(6)
41(6)
123(6)
269(7)io
61(7)
272(7)255
59(7)
264(8)
269(8)p1
61(8)
272(8)10
59(8)
264(9)
269(9)p2
61(9)
269(9)p1
42(9)
269(9)p1
59(9)
264(10)
269(10)a
61(10)
269(10)p2
45(10)
269(10)p1
59(10)
264(11)
269(11)b
61(11)
269(11)p1
42(11)
269(11)p2
59(11)
269(12)p1
61(12)
269(12)b
47(12)
269(12)p2
59(12)
269(13)p1
61(13)
269(13)soma
40(13)
269(13)p1
44(13)
269(13)p2
41(13)
59(13)
266(14)
269(14)p1
59(14)
125(15)
263(17)
269(17)ret_float
40(17)
267(17)
41(17)
123(17)
263(19)
269(19)a
61(19)
271(19)0x1p-1 / 0.500000
59(19)
263(20)
269(20)b
61(20)
272(20)253
43(20)
272(20)161
59(20)
263(21)
269(21)_c_
61(21)
269(21)a
43(21)
269(21)b
59(21)
263(22)
269(22)_2d
61(22)
269(22)c
47(22)
272(22)2
59(22)
266(23)
269(23)_2d
47(23)
272(23)2
269(23)d
59(23)
125(24)
267(26)
269(26)main
40(26)
41(26)
123(26)
262(27)
269(27)a
61(27)
265(27)
262(27)
91(27)
272(27)20
93(27)
59(27)
269(28)a
61(28)
273(28)Teste Lexico
59(28)
269(29)a
61(29)
273(29)Teste com string
59(29)
264(30)
269(30)b
61(30)
272(30)10
59(30)
264(31)
269(31)c
61(31)
272(31)25
59(31)
260(32)
40(32)
269(32)c
62(32)
269(32)b
41(32)
123(32)
269(33)b
61(33)
269(33)b
42(33)
272(33)3
59(33)
125(34)
261(35)
123(35)
269(36)b
61(36)
269(36)c
59(36)
125(37)
268(38)
40(38)
269(38)c
257(38)
272(38)0
41(38)
123(38)
260(39)
40(39)
269(39)c
258(39)
269(39)b
41(39)
123(39)
269(40)c
61(40)
269(40)c
45(40)
272(40)1
59(40)
125(41)
125(42)
125(43)
269(47)a
269(48)a
42(49)
47(49)
269(49)a
269(50)a
42(51)
42(51)
42(51)
42(51)
42(51)
42(51)
47(51)
269(51)a
269(52)a
42(52)
42(52)
42(52)
267(54)
269(54)main
40(54)
41(54)
123(54)
262(55)
269(55)a
61(55)
265(55)
262(55)
91(55)
272(55)20
93(55)
59(55)
269(56)a
61(56)
273(56)Testando primeira \ string
59(56)
269(57)a
61(57)
273(57)Testando em 
 duas linhas
59(57)
269(58)a
61(58)
273(58)
 Segunda linha
59(58)
269(59)a
61(59)
273(59)Primeira 
 Segunda 
 Terceira
59(59)
269(60)a
61(60)
273(60)Testando	tab1	tab2
59(60)
269(61)a
61(61)
273(61)Testando			 3 tabs
59(61)
269(62)a
61(62)
273(62)/***/
59(62)
269(63)a
61(63)
273(63)/***/
59(63)
125(64)
END
